src/test/java/com/google/javascript/gents/singleTests/comments.ts

```diff
@@ -36,3 +36,17 @@ const x;
 
 /** @export */
 let m: number = 4;
+let a = function() {
+  return {b: function() {}};
+};
+let c = function() {};
+
+// comment before GETPROP
+
+// comment in GETPROP
```

Comment:
This comment is above `a()` after migration which is not ideal.
This is due to the limitation that comments are mapped from nodes.